modules/react/tabs/lib/TabsOverflowButton.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+
+import {chevronDownSmallIcon} from '@workday/canvas-system-icons-web';
+import {
+  createComponent,
+  useModelContext,
+  createHook,
+  composeHooks,
+  subModelHook,
+} from '@workday/canvas-kit-react/common';
+import {SystemIcon} from '@workday/canvas-kit-react/icon';
+import {useOverflowTarget} from './overflow';
+import {useMenuTarget} from './menu';
+
+import {TabsModelContext} from './Tabs';
+import {TabsModel} from './useTabsModel';
+import {StyledTabItem} from './TabsItem';
+
+export interface OverflowButtonProps {
+  /**
+   * The label text of the Tab.
+   */
+  children: React.ReactNode;
+  /**
+   * Optionally pass a model directly to this component. Default is to implicitly use the same
+   * model as the container component which uses React context. Only use this for advanced use-cases
+   */
+  model?: TabsModel;
+}
+
+export const TabsOverflowButton = createComponent('button')({
+  displayName: 'Tabs.OverflowButton',
+  Component: ({model, children, ...elemProps}: OverflowButtonProps, ref, Element) => {
+    const localModel = useModelContext(TabsModelContext, model);
+
+    const props = useTabsOverflowButton(localModel, elemProps, ref);
+
+    return (
+      <StyledTabItem hasIcon={true} type="button" as={Element} {...props}>
```

Comment:
do we want hadIcon true always set here?

Reply:
Yes. The Overflow button has an icon. This can still be overridden.